applications/nrf5340_audio/src/bluetooth/le_audio_cis_headset.c

```diff
@@ -4,3 +4,281 @@
+
+	return 0;
+}
+
+static int stream_cap_release_cb(struct bt_audio_stream *stream)
+{
+	LOG_INF("Release: stream %p", (void *)stream);
+	return 0;
+}
+static struct bt_audio_capability_ops stream_cap_codec_ops = {
+	.config = stream_cap_config_cb,
+	.reconfig = stream_cap_reconfig_cb,
+	.qos = stream_cap_qos_cb,
+	.enable = stream_cap_enable_cb,
+	.start = stream_cap_start_cb,
+	.metadata = stream_cap_metadata_cb,
+	.disable = stream_cap_disable_cb,
+	.stop = stream_cap_stop_cb,
+	.release = stream_cap_release_cb,
+};
+
+static void stream_recv(struct bt_audio_stream *stream, const struct bt_iso_recv_info *info,
+			   struct net_buf *buf)
+{
+	static uint32_t recv_cnt;
+	bool bad_frame = false;
+
+	if (receive_cb == NULL) {
+		LOG_ERR("The RX callback has not been set");
+		return;
+	}
+
+	if (!(info->flags & BT_ISO_FLAGS_VALID)) {
+		bad_frame = true;
+	}
+	//TODO: change the payload size according to the ISO setting
+	receive_cb(buf->data, 120, bad_frame, info->ts);
+
+	recv_cnt++;
+	if ((recv_cnt % 1000U) == 0U) {
+		LOG_INF("Received %u total ISO packets", recv_cnt);
+	}
+}
+
+static void stream_stop(struct bt_audio_stream *stream)
+{
+	int ret;
+	struct event_t event;
+
+	LOG_INF("Stop: stream %p", (void *)stream);
+
+	event.event_source = EVT_SRC_LE_AUDIO;
+	event.le_audio_activity.le_audio_evt_type = LE_AUDIO_EVT_NOT_STREAMING;
+
+	ret = ctrl_events_put(&event);
+	ERR_CHK(ret);
+}
+static struct bt_audio_stream_ops stream_ops = {
+	.recv = stream_recv,
+	.stopped = stream_stop
+};
+
+static struct bt_audio_capability caps = {
+	.type = BT_AUDIO_SINK,
+	.pref = BT_AUDIO_CAPABILITY_PREF(
+			BT_AUDIO_CAPABILITY_UNFRAMED_SUPPORTED,
+			BT_GAP_LE_PHY_2M, 0x02, 10, 10000, 10000,
+			10000, 10000),
+	.codec = &lc3_codec,
+	.ops = &stream_cap_codec_ops,
+};
+
+static void connected(struct bt_conn *conn, uint8_t err)
+{
+	char addr[BT_ADDR_LE_STR_LEN];
+
+	bt_addr_le_to_str(bt_conn_get_dst(conn), addr, sizeof(addr));
+	if (err != 0) {
+		default_conn = NULL;
+		return;
+	}
+
+	LOG_INF("Connected: %s", addr);
+	default_conn = bt_conn_ref(conn);
+}
+
+static void disconnected(struct bt_conn *conn, uint8_t reason)
+{
+	char addr[BT_ADDR_LE_STR_LEN];
+
+	if (conn != default_conn) {
+		return;
+	}
+
+	bt_addr_le_to_str(bt_conn_get_dst(conn), addr, sizeof(addr));
+
+	LOG_INF("Disconnected: %s (reason 0x%02x)", addr, reason);
+
+	bt_conn_unref(default_conn);
+	default_conn = NULL;
+}
+
+BT_CONN_CB_DEFINE(conn_callbacks) = {
+	.connected = connected,
+	.disconnected = disconnected,
+};
+
+int le_audio_config_get(void)
+{
+	return 0;
+}
+
+int le_audio_volume_up(void)
+{
+	return 0;
+}
+
+int le_audio_volume_down(void)
+{
+	return 0;
+}
+
+int le_audio_volume_mute(void)
+{
+	return 0;
+}
+
+int le_audio_play(void)
+{
+	return 0;
+}
+
+int le_audio_pause(void)
+{
+	return 0;
+}
+
+int le_audio_send(uint8_t const *const data, size_t size)
+{
+	return 0;
+}
+
+void le_audio_enable(le_audio_receive_cb recv_cb)
+{
+	int ret;
+
+	receive_cb = recv_cb;
```

Comment:
Create an initialize function like this one here?:
https://github.com/erikrobstad/sdk-nrf/pull/4/commits/498a8ba4d784d40f91af63bf5c9b46ea867baafb#diff-b626e4e02deab2cd281e657fa26dd96a01c63f02ec1e21660ce2c57657030ba0R197